contents/docs/features/retention.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Retention
+sidebar: Docs
+showTitle: true
+---
+<br>
+PostHog provides a Retention table to show you how many users return on subsequent days after visiting the website.
+<br>
+<br>
+
+![Retention Table](../../images/retention-table.png)
+
+
```

Comment:
We should perhaps explain exactly how to read the above graph as it is a little complex.

Reply:
Got it!